client/src/commands/connectCommand.ts

```diff
@@ -66,6 +68,14 @@ export async function connect(gatewayRegistryEntry: FabricGatewayRegistryEntry,
         connection = FabricConnectionFactory.createFabricRuntimeConnection(runtime);
         runtimeData = 'managed runtime';
 
+        const fabricWalletRegistry: FabricWalletRegistry = FabricWalletRegistry.instance();
+        if (!fabricWalletRegistry.exists(runtime.getName())) {
+            const fabricWalletRegistryEntry: FabricWalletRegistryEntry = new FabricWalletRegistryEntry();
+            fabricWalletRegistryEntry.walletPath = wallet.getWalletPath();
+            fabricWalletRegistryEntry.name = runtime.getName();
+            await fabricWalletRegistry.add(fabricWalletRegistryEntry);
```

Comment:
do we really want the local runtime wallet in the registry list in the user settings?